### PR TITLE
Mention unstable features in /_matrix/client/versions

### LINKED ISSUE #1696: Mention unstable features in /_matrix/client/versions
[Spec link](https://adoring-einstein-5ea514.netlify.app/client-server/#get_matrixclientversions)

Synapse currently does something like
```json
GET https://matrix.org/_matrix/client/versions
{
  "versions": [
    "r0.0.1",
    "r0.1.0",
    "r0.2.0",
    "r0.3.0",
    "r0.4.0",
    "r0.5.0",
    "r0.6.0"
  ],
  "unstable_features": {
    "org.matrix.label_based_filtering": true,
    "org.matrix.e2e_cross_signing": true,
    "org.matrix.msc2432": true,
    "uk.half-shot.msc2666": true,
    "io.element.e2ee_forced.public": false,
    "io.element.e2ee_forced.private": false,
    "io.element.e2ee_forced.trusted_private": false
  }
}
```

It would be good if we could get enabled MSCs listed in here so things like Cerulean could dynamically decide if the homeserver being used has enough support to work.

### PR DESCRIPTION
**Pull request by sumitks866**
Fixes #1696 
### Pull Request Checklist

* [ ] Pull request includes a [sign off](https://github.com/matrix-org/dendrite/blob/master/docs/CONTRIBUTING.md#sign-off)

Signed-off-by: `Sumit Kumar Singh sumitks866@gmail.com`

### CODE REVIEW

File: clientapi/routing/routing.go [hunk 1]
```diff
@@ -68,14 +68,17 @@ func Setup(
...
+					UnstableFeatures map[string]bool `json:"unstable_features"`
+				}{Versions: []string{
 					"r0.0.1",
 					"r0.1.0",
 					"r0.2.0",
 					"r0.3.0",
 					"r0.4.0",
 					"r0.5.0",
 					"r0.6.1",
+				},UnstableFeatures: map[string]bool{
```
Comment (neilalexander) [inline clientapi/routing/routing.go:80]: It might make sense to only list MSCs here that are enabled in the config?

Comment (sumitks866) [reply clientapi/routing/routing.go:80]: Hi! Is there any variable which stores list of enabled MSCs. I am still understanding the codebase.
Thank You

Comment (neilalexander) [reply clientapi/routing/routing.go:80]: The `routing.Setup` function in `clientapi/routing/routing.go` has access to `cfg *config.ClientAPI`, which contains access to some of the relevant config for the client API.

However, there's no reference in that struct to the `MSCs` struct from `config_mscs.go`, so you may need to update `setup/config/config_clientapi.go` first so there's a reference to the `MSCs` struct.

**Comment by sumitks866** [comment]
@neilalexander .  I hope I have not messed up. Do I need to worry about failing checks?

**Review by Kegsay — APPROVED** [review]
> clientapi/routing/routing.go:66: File is not `goimports`-ed (goimports)

otherwise LGTM!

**Review by neilalexander — APPROVED** [review]

**Pull request merged**